presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraType.java

```diff
@@ -169,7 +169,7 @@ public static CassandraType getCassandraType(DataType.Name name)
             case VARINT:
                 return VARINT;
             default:
-                return null;
+                throw new PrestoException(NOT_SUPPORTED, "Unsupported column type: " + name);
```

Comment:
Can we ignore unsupported columns, like we do in JDBC connectors?

Also, if this method cannot return `null` now, remove the null check here: 
https://github.com/prestosql/presto/blob/1eb185af166b1cda13c9cf40b1690aba36e21b83/presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/NativeCassandraSession.java#L324

Reply:
I see. Let me check the detail.

Reply:
Updated.